python/setup.py

```diff
@@ -50,8 +50,8 @@
     if 'clean' in sys.argv[1:]:
         sys.exit(0)
     print("*************************************************************************")
-    print("Please run `configure` and `make` from the root of the source tree to    ")
-    print("generate version.py                                                      ")
+    print("Please run `configure` and `make` from the root of the source tree to")
+    print(f"generate {bifrost_version_file}")
```

Comment:
Oops, I guess the f-string syntax hoses 2.7 for no particularly compelling reason. Easy to avoid, but are we ready to cut off 2.7 entirely and remove it from CI?

Reply:
Someday we should drop support for 2.7 but I'm not sure that I'm entirely ready to do that now.